### DISCUSSION TITLE
Unable to Access Pimcore Documentation URLs.

### DISCUSSION THREAD

**Discussion by KasthuriAnandhan** [pimcore/pimcore#17476] [Q&A / Support] [answered]
Hi Team,
I am unable to access the Pimcore's  current version of documentation URLs. Is there any problems with that.
![image](https://github.com/user-attachments/assets/bbac68ad-eee9-43b5-89b0-e5d5efe8bb95)

@kingjia90 , @fashxp

**Comment by kingjia90** [comment]
Hi Katshuri,
yes, it seems currently down, thank you for reporting, we are looking on it and will let you know once it's up again

**Reply by kingjia90** [reply, marked as answer]
I am happy to announce that the docs are online again, thank you again

**Comment by KasthuriAnandhan** [comment]
Thankyou @kingjia90
